client/components/data/query-countries/index.jsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { isFetching } from 'state/countries/selectors';
+import { requestCountries } from 'state/countries/actions';
+
+class QueryCountries extends Component {
+	componentWillMount() {
```

Comment:
We're now recommending to do side-effects in `componentDidMount` and `componentDidUpdate`

See also https://gist.github.com/bvaughn/923dffb2cd9504ee440791fade8db5f9